modules/sqs/main.tf

```diff
@@ -3,6 +3,15 @@ locals {
   tags = merge(var.tags, { Deployment = var.prefix })
 }
 
+## SQS IAM access policy for metadata-queue.fifo SQS
+## ====================================================================================================
+data "aws_iam_policy_document" "metadata_queue_policy" {
+  statement {
+    actions   = ["sqs:*"] # todo: Lock down access to specific actions and resources.
+    resources = ["arn:aws:sqs:*"]
+    effect    = "Allow"
```

Comment:
we should make it more restricted in the future.

Reply:
Agreed. Added.